docs/search/query-development-guide.md

```diff
@@ -0,0 +1,186 @@
+Hypotheses of user expectations and search behaviour are validated by analysis.  Once validated, different query tools are deployed and tested in order to meet these expectations.  
+
+In cases where links to analysis or relevance testing are missing, this work is either pending or redundant. In cases where links to query tools are missing, this query type is served by the tools used for Concepts, subjects.
```

Comment:
> In cases where links to query tools are missing, this query type is served by the tools used for Concepts, subjects.

Is this suggesting that we aren't using query tools for concepts?

Reply:
The tools used for Concepts, subjects are the default generic tool set. Any hypotheses without query tools listed use the default, ie Concepts, subjects. Maybe it's just easier if I copy and paste the tools for each section.

Reply:
> default generic tool set

I'm not sure I know what that is?

Reply:
Right now, Contributors and Titles have special query configurations which are more involved than queries that are Concept, subjects.  For this reason, I'm saying the query for Concepts, subjects is the generic/default tool set because it's the one that returns the results if there are no matches for Contributor or Title fields.

Leave it with me. I'll drop the bit about cases where query tools are missing and instead paste in the same combo as for Concepts, subjects.